lib/rawQueries.js

```diff
@@ -3,17 +3,17 @@
 const Queries = {
     getPipelineUsageCountForJobTemplatesQuery: (
         tablePrefix = ''
-    ) => `SELECT COUNT(DISTINCT "pipelineId") as count, "templateId"
+    ) => `SELECT COUNT(DISTINCT "pipelineId") as count, "templateId", "id"
```

Comment:
Why do we need to sub group by `id`?
```suggestion
    ) => `SELECT COUNT(DISTINCT "pipelineId") as count, "templateId", "id"
```